test/test_schema.rb

```diff
@@ -6,22 +6,33 @@ def initialize(model)
   end
 
   def perform(ids)
+    sleep 1
```

Comment:
I left them in since there are so few tests, everything still runs in seconds, and we can verify that >= 1 second was spent in batch loading. It's mainly like this for ease for understanding the tests.